woocommerce/wc-admin
# Webpack: Update resolve path

This PR updates the [webpack `resolve.modules`](https://webpack.js.org/configuration/resolve/#resolve-modules) to include our `js/src` folder, which will let us import from folders/files with this as a base, like Calypso + `client`-relative paths. Now that we're adding components, we have a bunch of import paths like `../../components/card` – this will let us do `components/card` from anywhere and it maps correctly to `js/src/components/card`.

This also moves the `use-filters` HoC into a new folder `components/higher-order`.

To test

- Run `npm run dev` or `npm run build`, and make sure the dashboard builds correctly
